packages/jam/transition/externalities/accumulate-externalities.ts

```diff
@@ -394,8 +398,8 @@ export class AccumulateExternalities
     accumulateMinGas: ServiceGas,
     onTransferMinGas: ServiceGas,
     gratisStorage: U64,
+    serviceId: U64,
```

Comment:
```suggestion
    wantedServiceId: U64,
```
or `requestedServiceId: U64`